src/index.ts

```diff
@@ -28,8 +28,10 @@ const sanitizeSafe = (string: string, part: string) => {
 /**
  * Generate a storybook ID from a component/kind and story name.
  */
-export const toId = (kind: string, name: string) =>
-  `${sanitizeSafe(kind, 'kind')}--${sanitizeSafe(name, 'name')}`;
+export const toId = (kind: string, name?: string) =>
+  `${sanitizeSafe(kind, 'kind')}${
+    typeof name === 'string' ? `--${sanitizeSafe(name, 'name')}` : ''
```

Comment:
I think it's enough to check if `name` exists here. You can skip typecheking here. WDYT?

```js
name ? `--${sanitizeSafe(name, 'name')}` : ''
```

Reply:
Yeah, I thought so too, but there's a test case for throwing an exception should the argument (name) be empty string. Without typecheking, an empty string would never be passed to `sanitizeSafe` and the function will not throw. Wasn't sure, if I should keep this behavior (throwing an exception)... Then again, the whole issue here is making the name arg optional...

Reply:
I went ahead and allowed `''` as story. We're intentionally adding support for this, so the test had to change.